app-text/htmlrecode/htmlrecode-1.3.1-r1.ebuild

```diff
@@ -0,0 +1,42 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit toolchain-funcs
+
+DESCRIPTION="Recodes HTML file using a new character set"
+HOMEPAGE="https://bisqwit.iki.fi/source/htmlrecode.html"
+SRC_URI="https://bisqwit.iki.fi/src/arch/${P}.tar.bz2"
+
+KEYWORDS="~amd64 ~ppc ~x86"
+LICENSE="GPL-2"
+SLOT="0"
+IUSE=""
+
+PATCHES=( "${FILESDIR}/${P}-ar.patch" )
+HTML_DOCS=( README.html )
```

Comment:
and why would you like to increase complexity of the code installing `a single` html page but calling things twice rather than using dodoc once? This does not really make sense

Reply:
@Zlogene 
The reason i made it this way was to get the html file into the ```html``` directory...
But you're right, it's only one single file... I've changed it now and only use ```dodoc```